.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+name: Build
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  build_windows:
+    name: 🪟 Build Windows App
+    runs-on: windows-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          channel: stable
+      - run: flutter pub get
+      - run: flutter build windows --release
+      - name: Upload Windows App to release
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: build\windows\x64\runner\Release\quran_publisher_desktop.exe
+          tag: v1.0-${{ github.run_number }}
+          overwrite: true
+          body: "It's beta"
```

Comment:
Hard-coded value

Reply:
well we could change the tag. We can get the version from pubscpec.yaml. And for body, what you have in mind?